clindmri/scripts/fsl_bedpostx.py

```diff
@@ -57,10 +57,28 @@
   corresponding to the respective b-value.
 - The *.bvec files contain a 3x1 vector for each gradient, indicating the
   gradient direction.
-- The diffuson dta: ith volume in the data corresponds to a measurement
+- The diffuson data: ith volume in the data corresponds to a measurement
   obtained after applying a diffusion-sensitising gradient with a b-value given
-  by th ith entry in *.bval and a gradient direction given by the ith vector in
-  *.bvec
+  by the ith entry in *.bval and a gradient direction given by the ith vector
+  in *.bvec
+
+**Output files**
+
+For every model parameter a distribution of values is estimated.
+- mean_*samples.nii.gz files: the mean of this distribution is saved for every
+  parameter.
+- merged_*samples.nii.gz files: for the parameters used for tractography
+  (e.g. orientations), samples from these distributions are saved.
```

Comment:
This present documentation will be completed at the same time as tractography/fsl.py